src/utils/characters.tsx

```diff
@@ -5,62 +5,155 @@ import {
   AvatarEve,
   AvatarHal,
   AvatarRoy,
+  SplashAsh,
+  SplashDot,
+  SplashEve,
+  SplashHal,
+  SplashRoy,
   TextAsh,
   TextDot,
   TextEve,
   TextHal,
   TextRoy,
 } from "~/assets/characters/index.js";
 
-// TODO: Change these to SVGs once received from designer
-import splashHal from "~/assets/characters/hal.png";
-import splashAsh from "~/assets/characters/ash.png";
-import splashRoy from "~/assets/characters/roy.png";
-import splashEve from "~/assets/characters/eve.png";
-import splashDot from "~/assets/characters/dot.png";
+import {
+  AshBlink,
+  AshBotBusted,
+  AshBotWin,
+  DotBlink,
+  DotBotBusted,
+  DotBotWin,
+  EveBlink,
+  EveBotBusted,
+  EveBotWin,
+  HalBlink,
+  HalBotBusted,
+  HalBotWin,
+  RoyBlink,
+  RoyBotBusted,
+  RoyBotWin,
+  TransitionLinesAsh,
+  TransitionLinesDot,
+  TransitionLinesEve,
+  TransitionLinesHal,
+  TransitionLinesRoy,
+  TransitionLinesVoting,
+} from "~/assets/animations/index.js";
 
 export const getCharacterAvatar = (characterName: CharacterName) => {
+  const avatarMap = {
+    hal: <AvatarHal />,
+    ash: <AvatarAsh />,
+    roy: <AvatarRoy />,
+    eve: <AvatarEve />,
+    dot: <AvatarDot />,
+  };
+
+  return avatarMap[characterName];
+};
+
+export const getCharacterAnimation = (
+  characterName: CharacterName,
+  isBot?: boolean,
+  isSelected?: boolean,
+) => {
   switch (characterName) {
     case "hal":
-      return <AvatarHal />;
+      if (isSelected && isBot) {
+        return HalBotBusted;
+      }
+      if (isSelected && !isBot) {
+        return HalBlink;
+      }
+      if (!isSelected && isBot) {
+        return HalBotWin;
+      }
+      return HalBlink;
     case "ash":
-      return <AvatarAsh />;
+      if (isSelected && isBot) {
+        return AshBotBusted;
+      }
+      if (isSelected && !isBot) {
+        return AshBlink;
+      }
+      if (!isSelected && isBot) {
+        return AshBotWin;
+      }
+      return AshBlink;
```

Comment:
Keep it DRY. Make a function that takes `isSelected`, `isBot` `bustedIcon`, `blinkIcon`. `botWinIcon` and does all these ifs and then use it in each case

Reply:
Something like this maybe:
```TS
case "ash":
    if (isSelected) {
      return isBot ? AshBotBusted : AshBlink;
    } else {
      return isBot ? AshBotWin : AshBlink;
    }
```